pkgs/top-level/all-packages.nix

```diff
@@ -933,7 +933,7 @@ in
   };
 
   logseq = callPackage ../applications/misc/logseq {
-    electron = electron_11;
+    electron = electron_12;
```

Comment:
Can we use the latest?

Reply:
latest electron(electron-13.0.1) doesn't work, would hit the same issue as in #124434